packages/@aws-cdk/aws-rds/lib/cluster-ref.ts

```diff
@@ -54,6 +54,15 @@ export interface DatabaseClusterAttributes {
    */
   readonly securityGroupId: string;
 
+  /**
+   * Whether the imported security group allows all outbound traffic or not
+   *
+   * Unless set to `false`, no egress rules will be added to the security group.
+   *
+   * @default true
+   */
+  readonly securityGroupAllowsAllOutbound?: boolean;
```

Comment:
I feel like the right thing would have been to specify `ISecurityGroup` instead of a security group ID but this ship has sailed...

Reply:
How about deprecating in favor a `securityGroup: ISecurityGroup`?

Reply:
Yes!